lib/cmds/merge_cmds/export.ts

```diff
@@ -50,13 +61,62 @@ interface ExportMigrationOptions {
   outputFile?: string
 }
 
+export const callExportAppAction = async ({
+  api,
+  appDefinitionId,
+  createChangesetActionId,
+  exportActionId,
+  sourceEnvironmentId,
+  targetEnvironmentId,
+  spaceId
+}: {
+  api: PlainClientAPI
+  appDefinitionId: string
+  createChangesetActionId: string
+  exportActionId: string
+  sourceEnvironmentId: string
+  targetEnvironmentId: string
+  spaceId: string
+}) => {
+  let changesetRef: string
+
+  try {
+    changesetRef = await callCreateChangeset({
+      api,
+      appDefinitionId,
+      appActionId: createChangesetActionId,
+      parameters: {
+        sourceEnvironmentId,
+        targetEnvironmentId
+      },
+      spaceId,
+      environmentId: targetEnvironmentId
```

Comment:
Why is this always the `targetEnvironmentId` and not the default environment id picked up by the config?

Reply:
As discussed, I would suggest adding a comment (to the code) explaining why we use the targetEnvironment here.